core/src/main/scala/org/apache/spark/TaskContext.scala

```diff
@@ -177,6 +177,12 @@ abstract class TaskContext extends Serializable {
    */
   def getLocalProperty(key: String): String
 
+  /**
+   * CPUs allocated to the task.
+   */
+  @Since("3.1.3")
```

Comment:
this generally isn't something we would pull back since its a feature, I would expect this in 3.3, maybe 3.2 depending on where its at @gengliangwang thoughts?

Reply:
@tgravescs Thanks for the ping.
As we discussed in the dev list, no new features (except the ongoing major projects) can be merged into 3.2. 
Let's follow the policy and make it 3.3 here, which should happen around half a year later IIUC.

Reply:
sounds good, thanks